packages/web-twig/tests/components-fixtures/selectValidationState.twig

```diff
@@ -0,0 +1,36 @@
+<Select
+    id="select-success"
+    name="select-success"
+    label="Validation success"
+    validationText="Success message"
+    validationState="success"
+    isRequired
+>
+    <option value="" selected disabled>Placeholder</option>
+    <option value="1">Option 1</option>
+    <option value="2">Option 2</option>
+</Select>
+<Select
+    id="select-warning"
+    name="select-warning"
+    label="Validation warning"
+    validationText="Warning message"
+    validationState="warning"
+    isRequired
+>
+    <option value="" selected disabled>Placeholder</option>
+    <option value="1">Option 1</option>
+    <option value="2">Option 2</option>
+</Select>
+<Select
+    id="select-danger"
+    name="select-danger"
+    label="Validation danger"
+    validationText="{{ ['Warning validation message', 'Second warning validation message'] }}"
```

Comment:
this should also probably be a "danger" "text", but we don't really have to change it. Just FYI.

Reply:
fixed